tests/foreman/api/test_hostgroup.py

```diff
@@ -1060,7 +1060,6 @@ def test_negative_update_name(self):
                 self.assertEqual(hostgroup.read().name, original_name)
 
 
-@skip_if_bug_open('bugzilla', 1235377)
```

Comment:
This is not the right way to skip an entire class. This actually messes up with the class and return a function instead.

Removing this since the related bug is closed.